components/dialog.tsx

```diff
@@ -14,6 +14,7 @@ export const Dialog = ({
     <BlueprintDialog
       className={styles.dialog}
       isCloseButtonShown={false}
+      title=""
```

Comment:
This prevents a console warning that I was getting spammed by; can move in to separate PR if you want.